src/server/scripts/Outland/BlackTemple/boss_illidan.cpp

```diff
@@ -569,7 +569,10 @@ struct boss_illidan_stormrage : public BossAI
             case PHASE_DEMON:
             {
                 scheduler.CancelAll();
-                DoCastSelf(SPELL_SUMMON_SHADOW_DEMON, true);
+
+                ScheduleTimedEvent(30s, [&] {
+                    DoCastSelf(SPELL_SUMMON_SHADOW_DEMON, true);
+                }, 100s);
```

Comment:
It will repeat out of the shadow phase

Reply:
P5 Frenzy and Trap do the same thing. All Phase schedulers have a CancelAll at the start as well

Reply:
The trap doesn't seem to be working. It's not trapping Illidan.

Reply:
> The trap doesn't seem to be working. It's not trapping Illidan.

Open a proper report then. That has nothing to do with this PR.